README.md

```diff
@@ -79,14 +83,14 @@ Your answer...
 You clone yet another Tunr repo. Put the following commands in the correct order necessary to make the app run. Delete the one command that will not be used.
 
 ```
-$ rake db:drop
-$ rake db:create
-$ rails new . -d postgresql
-$ bundle install
 $ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
+$ cd moar-tunr # I added this (we must be in the project directory for the next commands to work)
+$ bundle install
+$ rails new . -d postgresql
```

Comment:
@jblakeman good point! I'm cool with that. Though in that case, your bundle install would come after this command, as there wouldn't be a Gemfile available yet :)